server/odc-migrate/src/main/resources/migrate/common/V_4_2_4_2__notification_policy_metadata.yaml

```diff
@@ -0,0 +1,991 @@
+kind: resource
+version: v2
```

Comment:
As this is a **metadata**, why do we need the same set of metadata for every organization? Can we just share the metadata in every organization?

Reply:
Sure, I mistakenly used the migration framework. I will fix this issue.

Reply:
I moved it into R4242NotificationPolicyMetaMigrate and notification-policy-metadata.yaml. This will only be executed once.